test/tvla_test.py

```diff
@@ -17,3 +19,24 @@ def test_help():
     tvla = TvlaCmd(Args('--help')).run()
     # Assert that a message is printed on stdout or stderr.
     assert(len(tvla.stdout()) != 0 or len(tvla.stderr()) != 0)
+
+
+def ttest_significant(ttest_trace) -> bool:
+    """Determine if a t-test trace contains a significant deviation from the mean."""
+    mean = np.mean(ttest_trace, axis=3)
+    stddev = np.std(ttest_trace, axis=3)
+    threshold = mean + 4.5 * stddev
```

Comment:
This should be replaced with just
`threshold = 4.5`

The data stored in `ttest.npy` are the t-test results, so we only want to check if any of the values is outside of the (-4.5, 4.5) interval

Reply:
Thanks, @vrozic! Fixed.